R/deps.R

```diff
@@ -152,7 +152,9 @@ dev_remote_type <- function(remotes = "") {
     } else {
       stop("Malformed remote specification '", x, "'", call. = FALSE)
     }
-    tryCatch(fun <- match.fun(paste0("install_", tolower(type))),
+    tryCatch(fun <- get(x = paste0("install_", tolower(type)),
```

Comment:
I'm not terribly consistent about enforcing this, but I'd prefer subsequent arguments to be indented just one extra level